docs/navigate.rst

```diff
@@ -43,7 +43,7 @@ View Node table
 
 1. Click the Visibility tab, and then click the :guilabel:`Node` table from the Table list.
 
-2. From the :guilabel:`Partition` list, click the partition.
+2. From the :guilabel:`Partition` list, click the partition. **All partition** partition is supported for the Node table in v2.9 and later.
```

Comment:
The **All partitions** option is supported for the Node table in v2.9 and later.